Etabs_Adapter/CRUD/Create/SectionProperty.cs

```diff
@@ -238,5 +241,167 @@ private void SetProfile(IProfile profile, string sectionName, IMaterialFragment
         }
 
         /***************************************************/
+
+
+        private bool GetFromDatabase(ISectionProperty bhSection)
+        {
+            if (EtabsSettings.DatabaseSettings.SectionDatabase == SectionDatabase.None)
+                return false;
+
+            string bhName = bhSection.Name;
+
+            bhName = bhName.ToUpper();
+            bhName = bhName.Replace(" ", "");
+
+            string[] sub = bhName.Split('X');
+
+            for (int i = 0; i < sub.Length; i++)
+            {
+                if (sub[i].EndsWith(".0"))
+                    sub[i] = string.Join("", sub[i].Take(sub[i].Length - 2).ToArray());
+            }
+
+            bhName = string.Join("X", sub);
+
+            string startOfName = string.Concat(bhName.TakeWhile(c => c < '0' || c > '9'));
+            string replace;
+            if (GetDictionary(EtabsSettings.DatabaseSettings.SectionDatabase).TryGetValue(startOfName, out replace))
+                bhName = bhName.Replace(startOfName, replace);
```

Comment:
Some quick comment lines on what is being done here would be helpful (know this is missing in big parts of the code, but always good to get better).
Can understand by reading the code, but a bit involved, so some quick lines like `//Remove any trailing '.0' from section name` and `Replace BHoM name convention with Etabs name convetions` would help.